addons/pointwatch/pointwatch.lua

```diff
@@ -323,6 +355,15 @@ function exp_msg(val,msg)
             -- If a merit point was not gained, 
             lp.current = math.min(lp.current,lp.tnm-1)
         end
+    elseif msg == 810 then
```

Comment:
I believe you also need msg == 809. From my testing I saw that 810 is the message for Exemplar points from a chain and 809 is the message for exemplar with no chain